build/Dockerfile.nginx

```
@@ -4,11 +4,15 @@ FROM nginx:1.25.3-alpine
 ARG NJS_DIR
 ARG NGINX_CONF_DIR
 
-RUN apk update && apk upgrade && apk add --no-cache libcap \
+RUN apk update && apk upgrade && apk add --no-cache libcap curl ca-certificates \
+    && printf "%s%s%s\n" "http://nginx.org/packages/mainline/alpine/v" `egrep -o '^[0-9]+\.[0-9]+' /etc/alpine-release` "/main" | tee -a /etc/apk/repositories \
```

```suggestion
    && printf "%s\n" "http://nginx.org/packages/mainline/alpine/v$(egrep -o '^[0-9]+\.[0-9]+' /etc/alpine-release)/main" >> /etc/apk/repositories \
```

This seems cleaner to me 😄 (but anyway we don't need `tee`)

---

I took this from the official NGINX docs, but yes, yours looks a lot cleaner 😂